src/idsk/components/footer-extended/footer-extended.js

```diff
@@ -0,0 +1,241 @@
+        // this.toggleClass($errorButton, 'idsk-display-none');
+
+        this.toggleClass($heartSymbol, 'idsk-heart-visible');
+
+        // let $infoQuestion = this.$module.querySelector('#idsk-info-question'); 
+        // $infoQuestion.innerText = 'Ďakujeme za Vašu spätnú väzbu   '
+
+    }
+    
+
+    handleNoButtonClick(e) {
+        let $errorOption = this.$module.querySelector('#idsk-error-form');
+        let $helpOption = this.$module.querySelector('#idsk-help-form');
+
+        let $feedbackQuestion = this.$module.querySelector('#feedback-question');
+
+        // this.addClass($errorOption, 'idsk-display-none');
+        // this.removeClass($errorOption, 'idsk-open');
+
+
+
+        //this.delay(2000).toggleClass($helpOption, 'idsk-display-none');
+        // window.setTimeout(function() {
+        //     this.toggleClass('idsk-display-none');
+        // }, 2000);
+        
+        this.toggleClass($feedbackQuestion, 'idsk-display-');
+
+        this.toggleClass($helpOption, 'idsk-display-none');
+
+        this.toggleClass($helpOption, 'idsk-open');
+    }
+
+    handleErrorButtonClick(e) {
+        let $errorOption = this.$module.querySelector('#idsk-error-form');
+        let $helpOption = this.$module.querySelector('#idsk-help-form');
+        let $feedbackQuestion = this.$module.querySelector('#feedback-question');
+        
+        
+        this.toggleClass($feedbackQuestion, 'idsk-display-');
+
+
+        this.addClass($helpOption, 'idsk-display-none');
+        this.removeClass($helpOption, 'idsk-open');
+
+        this.toggleClass($errorOption, 'idsk-display-none');
+        this.toggleClass($errorOption, 'idsk-open');
+
+
+        // this.toggleClass($writeUsOption, 'idsk-display-none');
+        // this.toggleClass($writeUsOption, 'idsk-open');
+        
+
+    }
+
+    handleCloseErrorFormButtonClick(e) {
+        let $errorOption = this.$module.querySelector('#idsk-error-form');
+        let $feedbackQuestion = this.$module.querySelector('#feedback-question');
+        //this.toggleClass($errorOption, 'idsk-feedback-hidden');
+       
+        this.toggleClass($feedbackQuestion, 'idsk-open');
+        this.toggleClass($feedbackQuestion, 'idsk-display-');
+        
+        this.toggleClass($errorOption, 'idsk-open');
+        this.toggleClass($errorOption, 'idsk-display-none');
+        //this.toggleClass($feedbackQuestion,'idsk-feedback-hidden' );    
+    }
+
+    handleCloseHelpFormButtonClick() {
+        let $helpOption = this.$module.querySelector('#idsk-help-form');
+        let $feedbackQuestion = this.$module.querySelector('#feedback-question');
+        
+        this.toggleClass($feedbackQuestion, 'idsk-open');
+        this.toggleClass($feedbackQuestion, 'idsk-display-');
+        this.toggleClass($helpOption, 'idsk-open');
+        this.toggleClass($helpOption, 'idsk-display-none');
+        //this.toggleClass($feedbackQuestion,'idsk-feedback-hidden' );  
+    }
+
+    addClass(node, className) {
```

Comment:
$elelent.classList.add("class-name"); robi to co chces... prida classu ak ju este el. nema